mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -211,11 +211,14 @@ final class EMLDAOptimizer extends LDAOptimizer {
     val docTopicDistributions: VertexRDD[TopicCounts] =
       graph.aggregateMessages[(Boolean, TopicCounts)](sendMsg, mergeMsg)
         .mapValues(_._2)
+    val prevGraph = graph
```

Comment:
I buy it, but where is graph persisted again?

Reply:
@zhengruifeng I'm still kind of curious, but trust your judgment so I will merge it.

Reply:
Thanks! I am on the holidy, so can no reply quickly.
Current change is following what Pregel does, 
I will go on looking into it, since the persistance in GraphX is a little complex.